.github/workflows/cmake.yml

```diff
@@ -243,6 +247,11 @@ jobs:
              # Limit parallel test jobs to prevent wine errors
             parallels-jobs: 3
 
+          - name: Ubuntu 16.04 Clang
+            os: ubuntu-16.04
+            compiler: clang-6.0
+            # note: github preinstalls compilers now, see https://github.com/actions/virtual-environments/pull/369/
```

Comment:
I'm not sure if the comment about preinstalling the compilers is necessary. It is good to know though.